packages/components/pie-card-container/src/defs.ts

```diff
@@ -47,4 +49,9 @@ export interface CardContainerProps {
      * What the interaction intent of the card should be such as anchor, button or regular div (default).
      */
      interactionType?: typeof interactionTypes[number];
+
+    /**
+     * An array of padding values from a to g.
```

Comment:
i think we need to update the description here, maybe just copy what we have in the props table

Reply:
Fixed